src/App.css

```diff
@@ -63,10 +69,25 @@ hr {
   list-style: none;
 }
 
+.Title {
+  /* width: 10.875rem; */
+  /* height: 1.5rem; */
+  margin-left: 10%;
+  font-family: Montserrat;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
 .School-of {
-  width: 7rem;
-  height: 1.125rem;
+  /* width: 7rem; */
+  /* height: 1.125rem; */
```

Comment:
## [OPTIONAL]

- [ ] Kindly remove commented code as they will make your code clean and clear